### DISCUSSION TITLE
Is it safe to install DSM 7.2.2-72803 on DS923+?

### DISCUSSION THREAD

**Discussion by tural-ali** [bb-qq/r8152#423] [Q&A] [answered]
Is it safe to install DSM 7.2.2-72803 on DS923+?

**Comment by bb-qq** [comment, marked as answer]
I updated my DS918+ to 7.2.2-72806 and confirmed no issue.

**Reply by bb-qq** [reply]
I think I have answered your question, so I will close this thread. If you have any questions, feel free to re-open it.
